### PR TITLE
(Role Base Implementation) Automatically setup role options

### PR DESCRIPTION
**Pull request by Pietrodjaowjao**

### CODE REVIEW

**Comment by Pietrodjaowjao** [comment]
its done now but missing neutrals

File: main.cs [hunk 1]
```diff
@@ -332,16 +333,27 @@ public static void LoadRoleClasses()
...
+                { CustomRoles.NiceMini, typeof(Mini) },
+                { CustomRoles.EvilMini, typeof(Mini) }
+            };
+
+
             foreach (var role in CustomRolesHelper.AllRoles.Where(x => x < CustomRoles.NotAssigned))
             {
-                Type roleType = role switch // Switch role to FatherRole (Double Classes)
+                Type roleType;
+                if (CustomRolesHelper.DuplicatedRoles.ContainsKey(role))
```
Comment (Ultradragon005) [inline main.cs:349]: jsut check if (CustomRolesHelper.DuplicatedRoles.TryGetValue(role, out roleType)) and then assign the roletype with that variable

Comment (Ultradragon005) [reply main.cs:349]: nvm

**Comment by Pietrodjaowjao** [comment]
damn mini is not appearing

**Review by Tommy-XL — APPROVED** [review]

**Pull request merged**